apis/dialapi/dialapi.yaml

```diff
@@ -427,8 +427,132 @@ paths:
             to contact us. In the meantime, try refreshing.
           content: {}
       deprecated: false
-      security: 
+      security:
         - bearer: []
+  /dialcode/v2/read:
+    post:
+      tags:
+        - DIAL APIs
+      summary: Get DIAL Code V2
+      description: |-
+        This API is associated with fetching/reading the DIAL code with context information
+          - The endpoint for **Get DIAL code V2** is `/dialcode/v2/read/{dialcode}`
+          - It points to knowledge-mw-service - `/v2/dialcode/read/{dialcode}`
+          - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty."
+      operationId: ReadPost
+      parameters:
+        - name: X-Channel-ID
+          in: header
+          description: It the Unique ID to identify the root organization to which the
+            user belongs
+          required: true
+          schema:
+            type: string
+        - name: Content-Type
+          in: header
+          description: "The Content Type entity is the media type of the resource.\n\
+            \ Possible media types can be:- \n   - Application/json"
```

Comment:
Line 454: There's double punctuation. Please remove the hyphen in _Possible media types can be:-_

Reply:
done